.github/workflows/verify_artifacts.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import argparse
+
+
+def create_artifact_paths(version):
+    artifacts = []
+    artifacts_base_path = "https://hyperledger.jfrog.io/hyperledger/besu-maven/org/hyperledger/besu"
+    # add to this list here to update the list of artifacts to check 
+    besu_paths = [
```

Comment:
@macfarla add to this list here to keep it fresh and it'll check jar, pom and module for each item